### PR TITLE
DTSCCI-332 display full name of user

### PR DESCRIPTION
**Pull request by rishikrsharma**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Screenshot attached for the Evidence of manual testing
- [ ] Functional, Smoke and API tests have been run locally
- [ ] Full gradle build and run tests with coverage have been completed (if needed)

Please remove this line and everything above and fill the following sections:

### Testing branch and instruction ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java [hunk 1]
```diff
@@ -102,4 +104,10 @@ protected CallbackResponse emptyCallbackResponse(CallbackParams callbackParams)
     protected CallbackResponse emptySubmittedCallbackResponse(CallbackParams callbackParams) {
         return SubmittedCallbackResponse.builder().build();
     }
+
+    public String getFullName(UserInfo userDetails) {
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java:108]: not sure this is the right place for this method.

Comment (rishikrsharma) [reply src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java:108]: both class extending this so why to duplicate or create another class

**Review by tiagreaney — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java (hunk 1 above)
Comment (velimd) [reply src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java:108]: fair. i was thinking more like json ignore in the data model place but not sure how GA project does stuff. Everything else LGTM.

**Review by velimd — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java (hunk 1 above)
Comment (rishikrsharma) [reply src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java]: @gmikerlemos  move this method to util class

**Review by velimd — APPROVED** [review]

**Pull request merged**